examples/closures.loxnb

```diff
@@ -0,0 +1 @@
+{"cells":[{"kind":2,"language":"lox","value":"// So far fails with: No variable 'outside' defined\n\nfun returnFunction(): () => void {\n    var outside = \"outside\";\n\n    fun inner(): void {\n        print outside;\n    }\n\n    return inner;\n}\n\nvar fn = returnFunction();\nfn();"},{"kind":2,"language":"lox","value":"// So far fails with: No variable 'exponent' defined\n\nfun power(exponent: number): (number) => number {\n\tfun applyPower(base: number): number {\n    \tvar current = 1;\n        for (var i = 0; i < exponent; i = i + 1) {\n        \tcurrent = current * base;\n        }\n        return current;\n    }\n    return applyPower;\n}\n\nvar cube = power(3);\n\nprint cube(1);\nprint cube(2);\nprint cube(3);\nprint cube(4);\nprint cube(5);\n"}]}
```

Comment:
Question: Is this designed as a test, or as an example? We already have this example in the `basic.lox` file. If you want to have this as a test, please write a unit test.

Reply:
I checked in my notebook files. Of course, those could be unit tests, will look into converting them.

Reply:
removed those notebook files